manticore/native/models.py

```diff
@@ -140,3 +172,56 @@ def strlen(state, s):
             ret = ITEBV(cpu.address_bit_size, byt == 0, offset, ret)
 
     return ret
+
+
+def strcpy(state: State, dst: Union[int, Expression], src: [int, Expression]) -> int:
+    """
+    strcpy symbolic model
+
+    Algorithm: Copy every byte from the src to dst until finding a byte that will is or must be '\000'
```

Comment:
```suggestion
    Algorithm: Copy every byte from the src to dst until finding a byte that will or must be '\000'
```

Reply:
I will fix this I forget to update it :)